docs/diagrams/ModelClassDiagram.puml

```diff
@@ -52,6 +53,9 @@ Person *--> Email
 Person *--> Address
 Person *--> "*" Tag
 
```

Comment:
This puml file is edited but the new diagram is not saved in `/images`.

Reply:
Thanks for the catch.